pkg/controller/job/job_controller.go

```diff
@@ -907,12 +906,10 @@ func (jm *Controller) manageJob(job *batch.Job, activePods []*v1.Pod, succeeded
 			}
 			diff -= batchSize
 		}
-		if err := errorFromChannel(errCh); err != nil {
-			return active, err
-		}
+		return active, "pods_created", errorFromChannel(errCh)
 	}
 
-	return active, nil
+	return active, "tracking", nil
```

Comment:
can you put the label values as constants in the metrics package we a comment for what they mean?